x-pack/plugins/security_solution/public/management/pages/policy/view/policy_list.tsx

```diff
@@ -54,8 +93,48 @@ export const PolicyList = memo(() => {
     return map;
   }, [endpointCount]);
 
-  const totalItemCount = data?.total ?? 0;
+  const totalItemCount = useMemo(() => data?.total ?? 0, [data]);
+
+  const policyListPath = useMemo(() => getPoliciesPath(search), [search]);
 
+  const backLink: PolicyDetailsRouteState['backLink'] = useMemo(() => {
+    return {
+      navigateTo: [
+        APP_UI_ID,
+        {
+          path: policyListPath,
+        },
+      ],
+      label: i18n.translate('xpack.securitySolution.policy.backToPolicyList', {
+        defaultMessage: 'Back to policy list',
+      }),
+      href: getAppUrl({ path: policyListPath }),
+    };
+  }, [getAppUrl, policyListPath]);
+
+  const handleCreatePolicyClick = useNavigateToAppEventHandler<CreatePackagePolicyRouteState>(
+    'fleet',
+    {
+      path: `/integrations/${
```

Comment:
We should use Fleet's exposed `pagePathGetters.add_integration_to_policy` service to get the URL instead of hardcoding it. See:  

https://github.com/elastic/kibana/blob/ef557af521fbd6ea844e743fb39b406eae7534d5/x-pack/plugins/fleet/public/constants/page_paths.ts#L143

Reply:
will do in the next pr!